src/wp-includes/compat.php

```diff
@@ -375,6 +377,44 @@ function is_iterable( $var ) {
 	}
 }
 
+if ( ! function_exists( 'str_starts_with' ) ) {
+	/**
+	 * Polyfill for str_starts_with() function added in PHP 8.
+	 * Performs a case-sensitive check indicating if needle is contained in haystack.
+	 *
+	 * @ticket 54377
+	 * @ignore
```

Comment:
- The `@ticket` annotation is not needed. 
- The `@ignore` annotation is not needed.